lib/checks/canFindDenominatorInNumerator.js

```diff
@@ -0,0 +1,35 @@
+const Node = require('../node');
+
+// Returns true if by adding a term you can simplify part of the function into an integer
+// e.g. (2x+1)/(2x+3) -> True
+// e.g. (2x+1)/(2x^2 + 3) -> False
+function canFindDenominatorInNumerator(node) {
+  if (!Node.Type.isOperator(node) || node.op !== '/' ) {
+    return false;
+  }
+  if (node.args.length !== 2) {
+    return false;
+  }
+
+  let numerator = node.args[0];
+  let denominator = node.args[1];
+  if (Node.Type.isParenthesis(numerator)) {
+    numerator = numerator.content;
+  }
+  if (Node.Type.isParenthesis(denominator)) {
+    denominator = denominator.content;
+  }
+
+  if (numerator.op !== '+') {
+    return false;
+  }
+  if (!('args' in denominator)) {
```

Comment:
line 23 is to check whether the numerator is the '+' op between nodes, so that's to check that there's an 'x + y' kind of thing going on, but now that you mention it, it's not actually required. cause we should be able to go from x/(x+1) -> (x+1)/(x+1) -1)/(x+1). I'll take this out.

line 26 is in there because tests were failing by having a constant in the bottom, i think a constant in the bottom is resolved best the original way.